apps/yara/CMakeLists.txt

```diff
@@ -14,22 +14,14 @@ message (STATUS "Configuring apps/yara")
 # Dependencies
 # ----------------------------------------------------------------------------
 
-# Disable older MSVC.
-if (MSVC AND MSVC_VERSION LESS 1800)
-    message (STATUS "  Visual Studio 2012 or newer required to build yara.")
-    return ()
-endif ()
-
-# Disable parallelism in MSVC as it supports only OpenMP 2.0.
-if (NOT MSVC)
```

Comment:
I am on a conference, so I cannot test anything on windows (and on vacation immediately after that). If you want to still have single-threaded visual builds for yara, can you send me diff (or make a PR) that does
  * remove `/openmp` from `SEQAN_CXX_FLAGS`
  * replace `-DSEQAN_HAS_OPENMP=1` with =0 in SEQAN_DEFINITIONS
?

Thanks!